js/reunion.js

```diff
@@ -0,0 +1,51 @@
+import { getMeeting } from './api/meetings.js';
+
+var url = new URL(window.location);
+
+const searchParams1 = new URLSearchParams(url.search);
+const param = searchParams1.get('meetingId');
+
+const meetInfo = async () => {
+  const meet = await getMeeting(param);
```

Comment:
Se podría usar `meeting` en el nombre de la variable, `meet` significa reunirse, no reunión
```suggestion
  const meeting = await getMeeting(meetingId);
```